src/styles/utility/_spacing.scss

```diff
@@ -0,0 +1,126 @@
+/*
+Generates utility classes for spacing (margin and padding)
+
+The convention is ".bdl-py-0"
```

Comment:
Maybe we should stick to SUIT style, and make the property classes below full words.
https://github.com/suitcss/suit/blob/master/doc/utilities.md
https://github.com/suitcss/suit/tree/master/packages (utils packages)

.bdl-u-marginLeft
.bdl-u-marginX

.bdl-u- prefix
margin, padding
null, X, Y, Top, Right, Bottom, Left
0,1,2... multiplier for grid units

Reply:
Maybe on the `u`... but I think compactness is an advantage here.